cmd/mic/main.go

```diff
@@ -10,14 +11,16 @@ import (
 )
 
 var (
-	kubeconfig  string
-	cloudconfig string
+	kubeconfig      string
+	cloudconfig     string
+	forceNamespaced bool
 )
 
 func main() {
 	defer glog.Flush()
 	flag.StringVar(&kubeconfig, "kubeconfig", "", "Path to the kube config")
 	flag.StringVar(&cloudconfig, "cloudconfig", "", "Path to cloud config e.g. Azure.json file")
+	flag.BoolVar(&forceNamespaced, "forceNamespaced", false, "Forces mic to namespace identities, binding, and assignment")
```

Comment:
nit: in the context of mic "mic to" is redundant, can be modified to
"Forces namespace identities, binding, and assignment"